src/umleditor/mvc_model/relation.py

```diff
@@ -108,4 +108,4 @@ def __eq__(self, other):
         if self._destination != other._destination:
             return False
         
-        return True
\ No newline at end of file
+        return self._type == other._type
```

Comment:
Either don't type check or add helper method that replaces this check wherever this overload is being used

Reply:
Fixed